### DISCUSSION TITLE
从其他机器copy的experiment log无法resume

### DISCUSSION THREAD

**Discussion by yuanjie-ai** [microsoft/nni#3796] [Q&A] [answered]
本机的experiment log可以resume
从其他机器copy的experiment log无法resume

**Comment by yuanjie-ai** [comment]
![image](https://user-images.githubusercontent.com/20265321/92081226-56f6b280-edf5-11ea-8323-42456b6100f3.png)

**Comment by yuanjie-ai** [comment]
why???
![image](https://user-images.githubusercontent.com/20265321/92091418-506f3780-ee03-11ea-91d2-2d09591d062d.png)

**Comment by SparkSnail** [comment, marked as answer]
NNI does not support packing search_space right now, you need to put a search_space.json file in correct location in new machine.
